iam/api-client/custom_roles_test.py

```diff
@@ -33,23 +33,21 @@
 
 @pytest.fixture(scope="module")
 def test_custom_role():
-    # section to create custom role to test policy updates.
-    custom_roles.create_role(
-        CUSTOM_ROLE_NAME,
-        GCLOUD_PROJECT,
-        CUSTOM_ROLE_TITLE,
-        CUSTOM_ROLE_DESCRIPTION,
-        CUSTOM_ROLE_PERMISSIONS,
-        CUSTOM_ROLE_STAGE,
-    )
-    yield CUSTOM_ROLE_NAME
-
-    # Delete the custom role
+    # This custom role is reused in read/update tests.
     try:
-        custom_roles.delete_role(CUSTOM_ROLE_NAME, GCLOUD_PROJECT)
+        custom_roles.create_role(
+            CUSTOM_ROLE_NAME,
+            GCLOUD_PROJECT,
+            CUSTOM_ROLE_TITLE,
+            CUSTOM_ROLE_DESCRIPTION,
+            CUSTOM_ROLE_PERMISSIONS,
+            CUSTOM_ROLE_STAGE,
+        )
     except googleapiclient.errors.HttpError:
-        print("Custom role already deleted.")
-
+        # Ignore error since we just reuse the same custom role.
```

Comment:
Can we make sure that we are ignoring the right error? Perhaps verify the message is correct and the snippet isn't failing for another reason?

Reply:
Sure! I'll make a change